bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -700,66 +702,57 @@
     */
    private int                   numberOfTimeSlices;
 
+   // ############################################# PHOTO  DATA #############################################
+
    /**
     * Time adjustment in seconds, this is an average value for all photos.
     */
    private int                   photoTimeAdjustment;
 
-   // ############################################# GEARS #############################################
-
    private int                   frontShiftCount;
+
    private int                   rearShiftCount;
 
-   // ############################################# RUNNING DYNAMICS #######################################
+   // ############################################# GEARS #############################################
 
    private short                 runDyn_StanceTime_Min;
    private short                 runDyn_StanceTime_Max;
-   private float                 runDyn_StanceTime_Avg;
 
+   // ############################################# RUNNING DYNAMICS #######################################
```

Comment:
There are too many formatting modifications

![tour-data-reformatted](https://user-images.githubusercontent.com/1283445/63174504-5a7c9780-c042-11e9-9794-29d5011b70f9.png)

Reply:
Arg, i dont know why Eclipse does that. I had the same problem with TourDatabase.java. I will fix it

Reply:
It's always a good idea to review the own changes before committing it